templates/show.html

```diff
@@ -0,0 +1,48 @@
+{% extends 'base.html' %}
+{% block head %}
+  {% load static %}
+  <script type="text/javascript" src="{% static "/src/external/jsxgraph.js" %}"></script>
+{% endblock %}
+{% block content %}
+    <div class="paste" id="application" role="application">
+        <main id="left-bar">
+          <header>
+            <img src="{% static "/assets/images/gridpaste-logo.svg" %}" alt="logo" />
+            <p>
+              <span class="red">Gridpaste</span> is an online math tool to share computations, transformations, and annotations on geometric structures in a coordinate plane. One starts with a clean board and places their geometric elements and performs actions on these elements in a recordable sequence. The sequence may then be saved and shared by a unique URL to colleagues or instructors.
+            </p>
+          </header> 
+          <section id="auth">
+            {% if user.is_authenticated %}
+            <div class="margin"><a href="/logout"> Log out ({{ user.username }}) </a></div>
```

Comment:
Avoid hard-coding urls. Use `{% url %}` instead.
https://docs.djangoproject.com/en/2.2/ref/templates/builtins/#url

This is why you named all your URL patterns, after all :)